src/base64.h

```diff
@@ -129,12 +129,6 @@ static size_t base64_encode(const char* src,
                             size_t slen,
                             char* dst,
                             size_t dlen) {
-  // We know how much we'll write, just make sure that there's space.
-  CHECK(dlen >= base64_encoded_size(slen) &&
-        "not enough space provided for base64 encode");
-
-  dlen = base64_encoded_size(slen);
```

Comment:
@bnoordhuis yeah. Got it. Thank you so much! I'm closing this PR then since it is wrong.